backend/src/main/java/ai/giskard/service/ProjectService.java

```diff
@@ -89,6 +92,11 @@ private void validateProjectKey(String projectKey) {
     public Project create(Project project, String ownerLogin) {
         String projectKey = project.getKey();
         validateProjectKey(projectKey);
+
+        if (LicenseUtils.isLimitReached(licenseService.getCurrentLicense().getProjectLimit(), (int) projectRepository.count())) {
+            throw new GiskardRuntimeException("Project limit is reached. You can upgrade your plan to create more.");
```

Comment:
Let's create a separate `LicenseValidationException` to handle these kind of errors. It'll allow us to handle it separately in `ExceptionTranslator` if needed